core/src/main/kotlin/no/strooped/view/screen/TaskScreen.kt

```diff
@@ -1,3 +1,102 @@
 package no.strooped.view.screen
 
-class TaskScreen
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.Screen
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.graphics.Texture
+import com.badlogic.gdx.math.Vector2
+import com.badlogic.gdx.scenes.scene2d.Stage
+import com.badlogic.gdx.scenes.scene2d.ui.Image
+import com.badlogic.gdx.utils.viewport.ScreenViewport
+import no.strooped.GameSingleton
+import no.strooped.StroopedController
+import no.strooped.TextureSizes
+import no.strooped.model.GameRoom
+import no.strooped.view.screen.components.ColorButton
+import no.strooped.view.screen.components.Label
+
+/**
+ * Uses https://otter.tech/an-mvc-guide-for-libgdx/ as inspiration
+ * */
+private val FONT_SIZE_LABEL_TEXT = TextureSizes.getScaledFontSize(3.0f)
+class TaskScreen(
+    private val controller: StroopedController,
+    private val gameRoom: GameRoom
+) : Screen {
+    private var ui: Stage = Stage(ScreenViewport())
+    private val backgroundPosition = Vector2(0f, 0f)
+    private val background: Image = Image(Texture("white.jpg"))
+    private var colorOptions: Array<ColorButton?> = arrayOfNulls(gameRoom.currentTask!!.possibleAnswers.size)
+    init {
+        print("Stuff")
+        // this init will disappear in future versions
+        /*Label label = new Label(labelText, skin);
+Pixmap labelColor = new Pixmap(labelWidth, labelHeight, Pixmap.Format.RGB888);
+labelColor.setColor(<your-color-goes-here>);
+labelColor.fill();
+label.getStyle().background = new Image(new Texture(labelColor)).getDrawable();*/
+        // TO DO for later
+        /*val toastFactory: Toast.ToastFactory = Builder()
+            .font(attr.font)
+            .build()*/
+        // https://github.com/wentsa/Toast-LibGDX - for toasts
+    }
+
+    override fun hide() {}
+
+    override fun show() {
+        background.setPosition(backgroundPosition.x, backgroundPosition.y)
+        background.setSize(Gdx.graphics.width.toFloat(), Gdx.graphics.height.toFloat())
+        ui.addActor(background)
+        val message = "Task ${gameRoom.currentTask!!.id}"
```

Comment:
Btw the task id is a uuid string and not a number. The best way to increment the task number, might be to have a taskCounter inside GameSingleton, which you can increment each time you change task

Reply:
done